radsat-sk/operation/crypt/encrypt.c

```diff
@@ -0,0 +1,58 @@
+#include "encrypt.h"
+
+int getPaddedBufSize(int plaintextLen);
```

Comment:
private functions should always use the `static` keyword, so they don't have global scope (need it here and at the full function definition)